api/MutationObserverInit.json

```diff
@@ -184,7 +185,8 @@
               }
             ],
             "edge": {
-              "version_added": true
+              "version_added": true,
```

Comment:
Since we know that anything in IE would be present in Edge 12 (the first Edge version), and we’re here and already modifying this data, can we also set this to “12”?